apps/client/src/app/components/rule/rule-settings-dialog/rule-settings-dialog.html

```diff
@@ -1,76 +1,126 @@
 <div mat-dialog-title>{{ data.rule.name }}</div>
 
 <div class="py-3" mat-dialog-content>
-  <div
-    class="w-100"
-    [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMin }"
-  >
-    <h6 class="mb-0">
-      <ng-container i18n>Threshold Min</ng-container>:
+  @if (
+    data.rule.configuration.thresholdMin && data.rule.configuration.thresholdMax
+  ) {
+    <div
+      class="w-100"
+      [ngClass]="{
```

Comment:
I think you can remove this. Because of the `@if` statement, the condition of `d-none` will never be true.